Source/Resources/Content/AadIam/MdmApplications/Microsoft Intune Enrollment.json

```diff
@@ -8,6 +8,7 @@
     "termsOfUseUrl": "https://portal.manage.microsoft.com/TermsofUse.aspx"
   },
   "appId": "d4ebce55-015a-49b5-a083-c84d1797ae8c",
+  "description":"What is it: Legacy version of 'Microsoft Intune', required in some tenants for Intune management. \n Learn More: https://github.com/MicrosoftDocs/azure-docs/issues/27017",
```

Comment:
**What does this do?**
Enables the legacy version of 'Microsoft Intune', which is required in some tenants for Intune management.

**Why should you use this?**
If you have clients that require the legacy version of 'Microsoft Intune.'
@metamenos this seems incorrect but curious to hear what you think lol 

**What is the end user impact?**
Users with legacy versions of Intune will be able to use Intune. 

**Learn more**
[Intune vs. Intune Enrollment](https://github.com/MicrosoftDocs/azure-docs/issues/27017)

Reply:
@david-simeon  I like it. This is one of those settings that Microsoft says you don't have to have, but if you don't have it sometimes doesn't work, so I think what you have is accurate.